Resources/translations/SonataTimelineBundle.de.xliff

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
-    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+    <file source-language="de" target-language="de" datatype="plaintext" original="file.ext">
```

Comment:
@OskarStark Yes, I trust you don't worry. ;-)

But I think this is not the good way.

Have to investigate.